projects/kubernetes-sigs/image-builder/patches/0013-Rename-Snow-node-image-to-reflect-appropriate-CAPI-p.patch

```diff
@@ -0,0 +1,26 @@
+From 5c4cac4d29f5952aeefc5eacb5c8b1dca51c8f8d Mon Sep 17 00:00:00 2001
+From: Abhay Krishna Arunachalam <arnchlm@amazon.com>
+Date: Fri, 10 Feb 2023 16:08:18 -0800
+Subject: [PATCH] Rename Snow node image to reflect appropriate CAPI provider
+
+Signed-off-by: Abhay Krishna Arunachalam <arnchlm@amazon.com>
+---
+ images/capi/packer/ami/packer.json | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/images/capi/packer/ami/packer.json b/images/capi/packer/ami/packer.json
+index 64fb3ce15..6e0b781c9 100644
+--- a/images/capi/packer/ami/packer.json
++++ b/images/capi/packer/ami/packer.json
+@@ -4,7 +4,7 @@
+       "access_key": "{{user `aws_access_key`}}",
+       "ami_description": "{{user `ami_description`}}",
+       "ami_groups": "{{user `ami_groups`}}",
+-      "ami_name": "capa-ami-{{user `build_name`}}-{{user `kubernetes_semver` | clean_resource_name}}-{{user `build_timestamp`}}",
++      "ami_name": "capas-ami-{{user `build_name`}}-{{user `kubernetes_semver` | clean_resource_name}}-{{user `build_timestamp`}}",
```

Comment:
maybe we could use `"{{user `ami_name`}}"` so we have full control over the name without changing the patch

Reply:
yeah I did have that originally